pretix_passbook/passbook.py

```diff
@@ -394,6 +394,11 @@ def generate_pass(self, order_position: OrderPosition):
                 gettext("Organizer contact"),
             )
         card.addBackField("orderCode", order.code, gettext("Order code"))
+        card.addBackField(
+            "purchase_date",
```

Comment:
It does not do anything AFAIK, but would it make sense to name it camelCase as the others are?

```suggestion
            "purchaseDate",
```